native/index.html

```diff
@@ -52,7 +237,18 @@
         <a href="/frontend-mini-challenges/react/dist/#/challenges">React Mini Challenges</a>
         <a href="/frontend-mini-challenges/vue/dist/#/">VueJs Mini Challenges</a>
       </div>
-
+      <label for="switch" >
+        <input type="checkbox" id="switch" class="switch">
+        <label for="switch" style="background-color: black; color: #080a0c;"></label>
+  </label>
+      <script>
+        const $html = document.querySelector("html");
+        const $checkbox = document.querySelector("#switch");
+        
+        $checkbox.addEventListener("change", function() {
+            $html.classList.toggle("light-mode");
+        });
+        </script>
```

Comment:
Also, take this script out and put it in a separate file. And then link that script in this file.